install_config/aggregate_logging.adoc

```diff
@@ -52,38 +52,24 @@ to manage container logs and prevent filling node disks.
 [[pre-deployment-configuration]]
 == Pre-deployment Configuration
 
-. Ensure that you have  deployed a router for the cluster.
-. Ensure that you have xref:../install_config/persistent_storage/index.adoc#install-config-persistent-storage-index[the
+. An Ansible playbook is available to deploy aggregated logging. You should
+familiarize yourself with the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installation and
```

Comment:
Yes, `playbooks/byo/openshift-cluster/openshift_logging.yml` is the right playbook to call when performing a logging install on a cluster that's already been installed.